osbs/api.py

```diff
@@ -321,6 +324,9 @@ def _create_build_directly(self, build_request, unique=None):
             builder_img['kind'] = 'DockerImage'
             builder_img['name'] = ref
 
+        build_json['metadata'].setdefault('annotations', {})
+        build_json['metadata']['annotations']['from'] = ','.join([build_kind, build_name])
```

Comment:
IMHO we should just use `json.dumps(builder_img)` here.

Reply:
Don't we run into the issue that kind and name in builder_image get changed if it's an ImageStreamTag?

Reply:
Well, right, it has to be done before that. Just commenting on the format of the annotation, that's all.

Reply:
ok I can do json.dumps